### PR TITLE
refactor(ui): flatten the DesignSystem hierarchy

### PR DESCRIPTION
**Pull request by rudolfs**
Dividing the design system into `Primitives` and `Components` didn't add much value and made things more confusing.
By having all components in one folder we only need one import:

`import { Copyable, Emoji, Icon, Identity, Modal } from "ui/DesignSystem";` 

First step towards https://github.com/radicle-dev/radicle-upstream/issues/1877.

### CODE REVIEW

**Review by geigerzaehler — COMMENTED** [review]
:construction_worker_man:

File: ui/DesignSystem/Funding/Pool/Receiver.svelte [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <script lang="typescript">
-  import { Copyable } from "../../../Component";
-  import { Button, Icon } from "../../../Primitive";
-  import type { Address, ReceiverStatus } from "../../../../src/funding/pool";
-  import { ellipsed } from "../../../../src/style";
+  import Copyable from "../../Copyable.svelte";
```
[the remark this answers is not in the document]
Comment (rudolfs) [reply ui/DesignSystem/Funding/Pool/Receiver.svelte]: 0dd04f1faa3b43dc16bd0ddf29500779f1938c46

File: ui/DesignSystem/SourceBrowser/RevisionSelector/Entry.svelte [hunk 2]
```diff
@@ -3,10 +3,9 @@
   import { BadgeType } from "ui/src/badge";
   import type { Branch, Tag } from "ui/src/source";
 
-  import IconBranch from "ui/DesignSystem/Primitive/Icon/Branch.svelte";
-  import IconLabel from "ui/DesignSystem/Primitive/Icon/Label.svelte";
-  import Spinner from "ui/DesignSystem/Component/Spinner.svelte";
-  import Badge from "ui/DesignSystem/Component/Badge.svelte";
+  import Badge from "../../Badge.svelte";
```
[the remark this answers is not in the document]
Comment (rudolfs) [reply ui/DesignSystem/SourceBrowser/RevisionSelector/Entry.svelte]: 5162c6ef55008368a9133a80b2b00d94667f67e6

File: ui/DesignSystem/SourceBrowser/Tree/File.svelte [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <script lang="typescript">
-  import { Icon } from "../../../Primitive";
-  import Spinner from "../../../Component/Spinner.svelte";
+  import Icon from "../../Icon";
```
[the remark this answers is not in the document]
Comment (rudolfs) [reply ui/DesignSystem/SourceBrowser/Tree/File.svelte]: 5162c6ef55008368a9133a80b2b00d94667f67e6

**Review by geigerzaehler — APPROVED** [review]

**Pull request merged**